crypto/digestible/test-utils/src/mock_merlin.rs

```diff
@@ -0,0 +1,23 @@
+use mc_crypto_digestible::DigestTranscript;
+use std::vec::Vec;
+
+// This is a mock of merlin, which doesn't hash anything, but captures
+// the inputs to calls to "append_bytes", so that we can inspect them,
+// write tests against them, etc.
+#[derive(Default, Clone, Debug)]
+pub struct MockMerlin {
```

Comment:
I have really been enjoying mockall for things like this, https://docs.rs/mockall/0.8.0/mockall/ , although what you've got here looks good, too.